### ISSUE TITLE
Allow authors to clone and include sections / resources

### ISSUE THREAD

**Issue by abziegler** [harvard-lil/h2o#575]
Currently authors can clone full casebooks only, and authors cannot pull sections or resources into their existing casebooks. 

We should enable authors to select, clone and include any casebook, section or resource in their existing casebook.

A challenge with this feature will be complexity, especially in search. Some initial ideas:

- Consider an "Include" action button, which lets logged_in user add a displayed casebook/section/resource to their own casebook. UX could be: (1) Click Include, (2) Select from dropdown of authored titles, (3) Render selected authored title, screen positioned at newly included item in the contents

- Consider an "advanced search" set of filters that lets user include relevant annotated cases/texts, sections, etc. with default search filters that exclude all but source cases and full casebooks

**Comment by holtchesley** [comment]
A version of this is staged and gated for superuser view only. 
Todo Still:
- [ ] Check out styles with many casebooks.
- [ ] Casebook target filtering?
- [ ] New Icon+Copy for Clone Section/Resource ( duplicating Clone casebook ).
